spec/models/category_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+describe Category do 
+it {should have_many (:videos)}
+describe '#recent_videos' do
+
+ it'return the  videos in reverse chronological order by created at' do
+ 	comedies = Category.create(name: "comedies")
```

Comment:
This file and the next one look like they are using tab characters instead of spaces.